solana/programs/example-native-token-transfers/src/instructions/admin.rs

```diff
@@ -204,6 +207,8 @@ pub struct RegisterTransceiver<'info> {
     pub payer: Signer<'info>,
 
     #[account(executable)]
+    // CHECK: Missing ownership check is OK here: Transceiver is intended to be an arbitrary
```

Comment:
@kcsongor I'm actually unsure about this annotation. Do we want to constrain the Transceiver in some way?
This is a privileged action so it's probably OK.